### PR TITLE
Add Run of Show to Portland 2020 prospectus

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 1]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+Hopin has a set of `sponsor information <https://support.hopin.to/en/articles/3574463-vendor-instructions>`_ that explains a bit more about the sponsorship setup on the platform.
+
+Sponsorship spaces
+------------------
+
+A quick overview of the spaces in the venue that are important:
+
+* The *main stage* is where the talks happen. This is also where Lightning talks will be given.
+* The *session area* is where the Unconference & Job Fair will happen. Sponsors will be assigned booths at the scheduled time for their job fair, and booths will be available for the duration of the conference.
```
Comment (mxsasha) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]: Thought: can we influence ordering of sessions in hopin's display? Showing job fair sessions at the top of the list? Or at least visible enough?

Comment (mxsasha) [reply docs/conf/portland/2020/sponsors/online-prospectus.rst]: "Sponsors will be assigned booths at the scheduled time for their job fair, and booths will be available for the duration of the conference." -> this seems contradictory. The job fair session is available for the scheduled time, the expo booth is available for the whole conference, right? Perhaps the part after the comma should move to the next bullet.

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 2]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+* The *expo area* is where sponsors will be able to have booths available. They are able to provide videos that will play 
+* The *networking area* is where you can chat with a random attendee to get to know people. There is not specific sponsorship usage of this area.
```
Comment (mxsasha) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* The *networking area* is where you can chat with a random attendee to get to know people. There is no specific sponsorship usage of this area.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 3]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
 ===========
 
-We are still working out the run of show for the virtual event.
+This Run of Show provides more context about the event and answers some common questions you may have. 
+Please let us know if there is any information missing that would be useful for you.
+
+Sponsorship schedule
+--------------------
+
+* **SUNDAY**: The conference online platform is open. You are welcome to hang look at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also welcome to lead a Writing Day project if your documentation is open source.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* **SUNDAY**: The conference online platform is open. You are welcome to hang out at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also welcome to lead a Writing Day project if your documentation is open source.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 4]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+* **MONDAY**: The conference platform opens at 8am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the main conference talks, when sponsors tables will need to be staffed. The conference will run until around 5pm.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]: In the main schedule I have the booths opening later, should I change that?

Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* **MONDAY**: The conference platform opens at 8am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the main conference talks, when sponsor booths will need to be staffed. The conference will run until around 5pm.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 5]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+* **TUESDAY**: The Job Fair will be Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* **TUESDAY**: The Job Fair will be on Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon. 
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 6]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+* **MONDAY**: The conference platform opens at 8am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the main conference talks, when sponsors tables will need to be staffed. The conference will run until around 5pm.
+
+* **TUESDAY**: The Job Fair will be Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon. 
+
+See the :doc:`full schedule </conf/{{ shortcode }}/{{ year }}/schedule>` for exact timing details.
+
+Sponsorship platform
+--------------------
+
+We will be using `Hopin <https://hopin.to/>`_ as our online conference platform. It allows a number of unique spaces for attendees during the conference, and we hope it will allow a good amount of interaction between attendees and sponsors.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
We will be using `Hopin <https://hopin.to/>`_ as our online conference platform. It has multiple unique spaces for attendees during the conference, and we hope it will allow for a good amount of interaction between attendees and sponsors.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 7]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+**We will open the online platform 1 week prior to the conference starting**. At that point, you will be able to login and look around the platform, but it won't become fully active until the Sunday of the conference.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]: This isn't accurate, because the only thing they'll be able to do is see the date/time of the event and set up their user accounts. Maybe we can reword it to reflect that?

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 8]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+Hopin has a set of `sponsor information <https://support.hopin.to/en/articles/3574463-vendor-instructions>`_ that explains a bit more about the sponsorship setup on the platform.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
Hopin has a set of `sponsor documentation <https://support.hopin.to/en/articles/3574463-vendor-instructions>`_ that explains a bit more about the sponsorship setup on the platform.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 9]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+Sponsorship spaces
+------------------
+
+A quick overview of the spaces in the venue that are important:
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
A quick overview of the important spaces in the "venue":
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst (hunk 1 above)
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* The *Sessions area* is where the Unconference & Job Fair will happen. Sponsors will be assigned sessions at the scheduled time for their job fair "tables", and Expo booths will be available for the duration of the conference.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 10]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+* The *main stage* is where the talks happen. This is also where Lightning talks will be given.
+* The *session area* is where the Unconference & Job Fair will happen. Sponsors will be assigned booths at the scheduled time for their job fair, and booths will be available for the duration of the conference.
+* The *expo area* is where sponsors will be able to have booths available. They are able to provide videos that will play
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* The *Expo area* is where sponsors will be able to have booths available. You can provide videos that will play whenever attendees enter your booth, as well as chat or video-call directly with attendees.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst (hunk 2 above)
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
* The *Networking area* is where you can chat with a random attendee to get to know people. There is no specific sponsorship usage of this area.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 11]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+* The *expo area* is where sponsors will be able to have booths available. They are able to provide videos that will play 
+* The *networking area* is where you can chat with a random attendee to get to know people. There is not specific sponsorship usage of this area.
+
+Sponsorship events
+------------------
+
+Job Fair
+~~~~~~~~
+
+On Tuesday morning we hold our Job Fair,
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst:206]:
```suggestion
On Tuesday morning we hold our Job Fair from 9:45AM-12:45PM,
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 12]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+which is a wonderful place to connect with our over 500 attendees.
+Many of them are looking for jobs now or will be in the near future,
+so it's a great chance to talk more about your company culture and open positions.
+
+**Logistics**: You will be assigned a session in the *session area* where you can engage with attendees and answer questions. We recommend that you answer answer general questions in the main room and then break off into private calls or chat to talk in more depth to specific people.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
**Logistics**: You will be assigned a session in the *Sessions area* where you can engage with attendees and answer questions. The session will have a max capacity of 10 participants (for video/audio), which means it'll serve as a "lobby/reception" area for your session. We recommend that you answer general questions in the main session and then break off into private calls or chats to talk in more depth with specific people. 
```

Comment (thatdocslady) [reply docs/conf/portland/2020/sponsors/online-prospectus.rst]: I just threw the number 10 as a start, maybe we want more/fewer?

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 13]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+Many of them are looking for jobs now or will be in the near future,
+so it's a great chance to talk more about your company culture and open positions.
+
+**Logistics**: You will be assigned a session in the *session area* where you can engage with attendees and answer questions. We recommend that you answer answer general questions in the main room and then break off into private calls or chat to talk in more depth to specific people. 
+
+Sponsor booths
+~~~~~~~~~~~~~~
+
+Sponsor booths are only available for our larger sponsors.
+You will be given a booth that is in the *expo area* of the online platform.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
You will be given a booth that is in the *Expo area* of the online platform.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 14]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+Most of the attendee focus will come during breaks between talks and at the beginning and end of the day.  
+
+**Logistics**: You can record a video to play at your booth when it isn't staffed, but we generally recommend having someone with the expo hall tab open to answer any questions that attendees might have. We recommend that you answer answer general questions in the main room and then break off into private calls or chat to talk in more depth to specific people.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
**Logistics**: You can record a video to play at your booth when attendees enter the booth, but we generally recommend having someone with the Expo hall tab open to answer any questions that attendees might have. We also recommend that you answer general questions in the main booth and then break off into private calls or chat to talk in more depth to specific people. 
```

Comment (thatdocslady) [reply docs/conf/portland/2020/sponsors/online-prospectus.rst]: How do we plan to request booth information from sponsors? We need to populate the fields in the Expo instance in hopin before the conference starts. I can make a google form if you need help with that...

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 15]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+Most of the attendee focus will come during breaks between talks and at the beginning and end of the day.  
+
+**Logistics**: You can record a video to play at your booth when it isn't staffed, but we generally recommend having someone with the expo hall tab open to answer any questions that attendees might have. We recommend that you answer answer general questions in the main room and then break off into private calls or chat to talk in more depth to specific people. 
+
+Writing day
+~~~~~~~~~~~
+
+On Sunday we hold our Writing Day.
+This is a place where the community gathers to get actual work done.
+This generally involved companies hosting a documentation sprint on some piece of documentation that is open source and needs improvements.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]:
```suggestion
This generally involved communities and organizations hosting a documentation sprint on some piece of documentation that is open source and needs improvements.
```

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 16]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+If you want to participate in the Writing Day,
+it helps to do a bit of work up front.
+The best way to prepare is to have a set of issues that you've already picked as "easy for beginners".
+Starting with these issues will make it much easier for people to start,
+and feel productive.
+Make sure you also have good installation instructions and other helpful beginners content as well.
+
+**Logistics**: You can email us before the conference, and we will add your project to the website as a project that attendees can work on. You can also introduce your project to attendees at 9:30 on Sunday during the Writing Day Introduction.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst]: Maybe reword to indicate the sign-up google spreadsheet?

File: docs/conf/portland/2020/sponsors/online-prospectus.rst [hunk 17]
```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
...
+
+How do I get the most out of my sponsorship?
+--------------------------------------------
+
+Come prepared to engage with our community, and to learn just as much as you teach. Engage with our event as attendees as well as sponsors. Send technical staff who can chat with people on the interesting things your company is doing, and get value from the vast amount of insight in the room. We do have some decision makers in the room, but soft sells will work better than hard sales in the environment we strive for.
+
+Who is my primary contact?
+--------------------------
+
+Eric Holscher will be your primary contact, but our team is available at sponsorship@writethedocs.org. If you have a time sensitive inquiry, please email the entire team to ensure a timely response.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/sponsors/online-prospectus.rst:248]: We should list the helpdesk info here

**Review by thatdocslady — COMMENTED** [review]
Nit-picks galore, otherwise looks great :D

**Comment by ericholscher** [comment]
Gonna merge this to send it out, I've addressed all the feedback 👍
